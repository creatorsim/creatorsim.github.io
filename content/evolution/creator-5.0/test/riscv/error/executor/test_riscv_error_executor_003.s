
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:
    li sp, 0
