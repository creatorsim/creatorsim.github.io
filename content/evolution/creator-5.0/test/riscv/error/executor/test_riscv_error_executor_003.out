

 Error found.
 Stack pointer cannot be placed in the text segment
cr[PC]:0x4; ir[x2,sp]:0x0; keyboard[0x0]:''; display[0x0]:''; 

